### PR TITLE
Feature #126 - Implement business get(RUD) view

### PR DESCRIPTION
**Pull request by datsiuk-ira**
Create RUD view for business and overwrite get detail method

### CODE REVIEW

File: beauty/api/views.py [hunk 1]
```diff
@@ -107,3 +108,16 @@ def get_object(self):
...
+    """Generic API for business GET, PUT and DELETE methods.
+    RUD - Retrieve, Update, Destroy"""
+    permission_classes = [IsAccountOwnerOrReadOnly]
+
+    queryset = Business.objects.all()
+    serializer_class = BusinessGetAllInfoSerializers
+
+    def get_info(self, instance):
+        """Method for getting all info about business"""
+        return get_object_or_404(self.queryset, id=instance.id)
```
Comment (pazuzu-ua) [inline beauty/api/views.py]: please, add a new line

**Review by pazuzu-ua — APPROVED** [review]

File: beauty/api/views.py [hunk 2]
```diff
@@ -107,3 +108,16 @@ def get_object(self):
         logger.info(f"{obj} was got for the user {user} (id={user.id})")
 
         return obj
+
+
+class BusinessDetailRUDView(RetrieveUpdateDestroyAPIView):
+    """Generic API for business GET, PUT and DELETE methods.
```
Comment (Serhii-Voloshyn) [inline beauty/api/views.py:125]: New line after class docstring needed.

File: beauty/api/views.py [hunk 3]
```diff
@@ -9,9 +9,10 @@
 from rest_framework.response import Response
 from rest_framework.reverse import reverse
 
-from .models import CustomUser, Order
+from .models import CustomUser, Order, Business
```
Comment (Misha86) [inline beauty/api/views.py]: if you import more than one class from one module should use such syntactic (class, class, ...)

**Review by Misha86 — CHANGES_REQUESTED** [review]
if you import more than one class from one module should use such syntactic (class, class, ...)

**Review by pazuzu-ua — APPROVED** [review]

**Review by OleksiiDatsiuk — APPROVED** [review]

**Review by Misha86 — APPROVED** [review]

**Pull request merged**